charts/base-cluster/templates/backup/velero.yaml

```diff
@@ -87,5 +87,23 @@ spec:
       volumeSnapshotLocation: []
     metrics:
       serviceMonitor:
+        enabled: true
```

Comment:
This should be conditional to prometheus being enabled, or are they smart and auto-detect prometheus?

Also, you should add a `dependsOn` for promtheus, if it's enabled

Reply:
I removed the check because you mentioned in an earlier comment, that the velero chart performs the check on its own

Reply:
Ah, oki dokes 👍

Reply:
But the `dependsOn` should be added nonetheless

Reply:
Yes

Reply:
Just for the documentation 
https://github.com/vmware-tanzu/helm-charts/blob/main/charts/velero/templates/prometheusrule.yaml#L1